website/blog/2023-06-25-databend-weekly-99.md

```diff
@@ -0,0 +1,112 @@
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Flink CDC
+
+Databend now supports Flink CDC (Change Data Capture), which allows you to capture and process real-time data changes from various sources using SQL-based queries. With Flink CDC, you can monitor and capture data modifications (inserts, updates, and deletes) happening in a database or streaming system and react to those changes in real-time.
+
+Databend's Flink SQL connector offers a connector that integrates Flink's stream processing capabilities with Databend. By configuring this connector, you can capture data changes from various databases as streams and load them into Databend for processing and analysis in real-time.
+
+If you are interested in learning more, please check out the resources listed below:
+
+- [Docs | Loading Data with Tools - Flink CDC](https://databend.rs/doc/load-data/load-db/flink-cdc)
+
+## Code Corner
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### Register External Tables with Databend's Python Binding
+
+Databend's Python Binding now supports Register External Table. You can use the following functions to register external tables in Databend:
+
+- `register_parquet`
+- `register_ndjson`
+- `register_csv`
+- `register_tsv`
+
+Here is an example of how to use the `register_parquet` function to register an external table named "ontime" in Databend:
+
+```python
+from databend import SessionContext
+
+ctx = SessionContext()
+
+ctx.register_parquet("ontime", "./ontime/", pattern = ".*.parquet")
+
+df = ctx.sql("select * from ontime limit 10").collect()
+
+print(df)
+```
+
+This code registers Parquet files located in the `./ontime/` directory with the pattern `.*.parquet` . You can then use the registered table name `ontime` in SQL queries to access the data.
+
+If you are interested in learning more, please check out the resources listed below:
+
+- [PR | feat(python): support register table functions](https://github.com/datafuselabs/databend/pull/11841)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- Added support for renaming columns with `ALTER TABLE [ IF EXISTS ] <name> RENAME COLUMN <col_name> TO <new_col_name>`.
+- Added support for column position to TSV format.
+- Added `system.background_jobs` and `system.background_tasks` tables.
+- Added http query deduplication via `X-DATABEND-DEDUPLICATE-LABEL` header.
+- Added support for distributed deletion.
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Implement Read-Only Mode for Databend Clusters
+
+Databend is planning to introduce a new feature called "Read-Only Mode" for Databend clusters. This will improve control and resource allocation for users.
```

Comment:
Databend plans to introduce a Read-Only Mode for clusters. This will provide users with more control over resource allocation.